toggl/mobileapp
# Run UI tests on cake

To test the UI tests:

`bash build.sh —target Tests.UI`

I had to change the target build architectures to x64 only; the NUnit runner fails otherwise. That won’t be a problem anyways.

One now needs to expose `TOGGL_UI_TEST_USERNAME ` and `TOGGL_UI_TEST_PASSWORD ` as environment variables. Take this into account when configuring the Bitrise flow.

This should remain `on-hold` until #387 is merged
